### PR TITLE
Linux packaging and Release tests

### LINKED ISSUE #195: Ability to test binaries after release
**Is your feature request related to a problem? Please describe.**
We need a way to test if the recently updated release binaries are working correctly.

**Describe the solution you'd like**
The best way, for now, would be to add another job in the `Release` workflow, where Kaytu will be installed and run. We can use `brew` to test the installation and compare the output of `kaytu version` to the current latest version(which would already be in the workflow).

### LINKED ISSUE #196: Linux packages
**Is your feature request related to a problem? Please describe.**
Have DNF/DNF/RPM, APT, and packaging for Linux to support installation on Redhat, Ubuntu/Debian systems

**Additional context**
This was proposed by Adnan

### PR DESCRIPTION
**Pull request by ADorigi**
fixes #196 
Adds `RPM` and `DEB` for `amd64` and `arm64` architectures.

fixes #195 
Original plan was to modify release workflow. To not break what is working, decided to not touch the main workflow. Instead, added a new workflow which tests the artifacts on `Release`.

### CODE REVIEW

**Comment by ADorigi** [comment]
I have added installation scripts for one line command to install `rpm` and `deb` packages

File: scripts/install_deb_amd64.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+apt update
+apt install -y wget curl
+
+DEB_FILE_URL=$(curl -s https://api.github.com/repos/kaytu-io/kaytu/releases/latest \
+| grep "browser_download_url.*amd64.deb" \
+| cut -d : -f 2,3 \
```
Comment (salehkhazaei) [inline scripts/install_deb_amd64.sh:8]: it would be cleaner if you use jq here

File: scripts/install_deb_arm64.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+apt update
+apt install -y wget curl
+
+DEB_FILE_URL=$(curl -s https://api.github.com/repos/kaytu-io/kaytu/releases/latest \
+| grep "browser_download_url.*arm64.deb" \
+| cut -d : -f 2,3 \
```
Comment (salehkhazaei) [inline scripts/install_deb_arm64.sh:8]: it would be cleaner if you use jq here

File: scripts/install_rpm_amd64.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+yum install -y wget curl
+
+RPM_FILE_URL=$(curl -s https://api.github.com/repos/kaytu-io/kaytu/releases/latest \
+| grep "browser_download_url.*amd64.rpm" \
+| cut -d : -f 2,3 \
```
Comment (salehkhazaei) [inline scripts/install_rpm_amd64.sh:7]: same

File: scripts/install_rpm_arm64.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+yum install -y wget curl
+
+RPM_FILE_URL=$(curl -s https://api.github.com/repos/kaytu-io/kaytu/releases/latest \
+| grep "browser_download_url.*arm64.rpm" \
```
Comment (salehkhazaei) [inline scripts/install_rpm_arm64.sh:6]: same

File: scripts/install_deb_arm64.sh (hunk 2 above)
Comment (ADorigi) [reply scripts/install_deb_arm64.sh]: now using jq

File: scripts/install_rpm_amd64.sh (hunk 3 above)
Comment (ADorigi) [reply scripts/install_rpm_amd64.sh]: now using jq

File: scripts/install_rpm_arm64.sh (hunk 4 above)
Comment (ADorigi) [reply scripts/install_rpm_arm64.sh]: now using jq

**Pull request merged**